api/institutions/serializers.py

```diff
@@ -270,7 +271,7 @@ def get_absolute_url(self, obj):
         )
 
 
-class InstitutionUserMetricsSerializer(JSONAPISerializer):
+class OldInstitutionUserMetricsSerializer(JSONAPISerializer):
```

Comment:
Maybe add docstring explain `New`/`Old` naming convention, dashboard flag names and purpose.